### PR TITLE
Enhancement: equery m maps the remote-id to the respective url.

### PR DESCRIPTION
**Pull request by siddhanthrathod**
Bug: https://bugs.gentoo.org/877519

### CODE REVIEW

File: pym/gentoolkit/equery/meta.py [hunk 1]
```diff
@@ -189,10 +189,43 @@ def _format_upstream_docs(docs):
 
     def _format_upstream_ids(ids):
         result = []
+        sites = {
```
Comment (thesamesam) [inline pym/gentoolkit/equery/meta.py:192]:
```suggestion
        remote_ids = {
```

File: pym/gentoolkit/equery/meta.py [hunk 2]
```diff
@@ -189,10 +189,43 @@ def _format_upstream_docs(docs):
...
+            "savannah-nongnu": "https://savannah.nongnu.org/projects/remote-id",
+            "sourceforge": "https://sourceforge.net/projects/remote-id/",
+            "sourcehut": "https://sr.ht/remote-id/",
+            "vim": "https://www.vim.org/scripts/script.php?script_id=remote-id",
+        }
         for id_ in ids:
-            site = id_[0]
-            proj_id = id_[1]
-            idstr = f"{site} ID: {proj_id}"
+            if id_[1] in sites:
```
Comment (thesamesam) [inline pym/gentoolkit/equery/meta.py:223]: Let's follow the Python "easier to ask forgiveness than permission" style here, by dropping that `if` and doing:
```suggestion
         	try:
         		site = ...
         		proj_id = ...
         	catch KeyError:
         		site = ...
         		proj_id = ...
```

File: pym/gentoolkit/equery/meta.py [hunk 3]
```diff
@@ -189,10 +189,43 @@ def _format_upstream_docs(docs):
...
+            "pecl": "https://pecl.php.net/package/remote-id",
+            "pypi": "https://pypi.org/project/remote-id/",
+            "rubygems": "https://rubygems.org/gems/remote-id/",
+            "savannah": "https://savannah.gnu.org/projects/remote-id",
+            "savannah-nongnu": "https://savannah.nongnu.org/projects/remote-id",
+            "sourceforge": "https://sourceforge.net/projects/remote-id/",
+            "sourcehut": "https://sr.ht/remote-id/",
+            "vim": "https://www.vim.org/scripts/script.php?script_id=remote-id",
+        }
         for id_ in ids:
```
Comment (thesamesam) [inline pym/gentoolkit/equery/meta.py:221]: I don't really love `id_`, maybe we should rename it to `id` or something instead.

File: pym/gentoolkit/equery/meta.py [hunk 4]
```diff
@@ -189,10 +189,43 @@ def _format_upstream_docs(docs):
...
-            site = id_[0]
-            proj_id = id_[1]
-            idstr = f"{site} ID: {proj_id}"
+            if id_[1] in sites:
+                site = sites[id_[1]].replace("remote-id", id_[0])
+                proj_id = id_[1]
+            else:
+                site = id_[0]
+                proj_id = id_[1]
+            idstr = f"{site} ( {proj_id} )"
```
Comment (thesamesam) [inline pym/gentoolkit/equery/meta.py:227]: Do we need the spaces around it?

```suggestion
            idstr = f"{site} ({proj_id})"
```

Comment (thesamesam) [reply pym/gentoolkit/equery/meta.py:227]: Also, result is directly below, so we could just inline it, I suppose:
	result.append(f"{...}")

File: pym/gentoolkit/equery/meta.py (hunk 1 above)
Comment (thesamesam) [reply pym/gentoolkit/equery/meta.py:192]: or maybe: `remote_id_map`? not sure

Comment (siddhanthrathod) [reply pym/gentoolkit/equery/meta.py]: remote_ids was available so went with that although there's one more var remoteids.. should I change it to remote_id_map ?

**Review by thesamesam — APPROVED** [review]

**Comment by thesamesam** [comment]
Nice job!

**Pull request closed without merging**